feature_PRESIDECMS-744 ::

cfmitrah opened the pull request:
- Review and enable more MIME Types for the Asset Manager

cfmitrah committed:
feature_PRESIDECMS-744 ::

- Review and enable more MIME Types for the Asset Manager

DominicWatson commented:
Hey guys, thanks for this. Could you also ensure that all the new file types also have a corresponding entry in:

```/preside/system/i18n/filetypes.properties```

This means that they will then render nicely in the file type picker, etc.

cfmitrah committed:
feature_PRESIDECMS-744 ::

- Added corresponding filetypes in  ( /preside/system/i18n/filetypes.properties ) file.

cfmitrah commented:
Added corresponding filetypes in  ( /preside/system/i18n/filetypes.properties ) file.

DominicWatson commented:
Garrrr, this got auto closed because the target branch has disappeared. Really sorry for this. You should be able to re-open from your branch though by targeting the `stable` branch.
